v4_abacus.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'v4_abacus'
-    spec.version                  = '1.7.31'
+    spec.version                  = ''1.7.31'
```

Comment:
hmm seems like an issue with the auto increment

Reply:
i probably fat fingered it when entering the value, good catch, updated d417bc77b118fe256edf130bd88679a1e50fa2b2